front/src/features/auth/useSignupForm.js

```
@@ -99,14 +97,19 @@ export function useSignupForm() {
 
   watch(
     () => form.verificationCode,
-    (newCode) => {
+    (newCode, oldCode) => {
+      if (newCode !== oldCode && errors.verificationCode) {
+        delete errors.verificationCode
+      }
+
       if (newCode.length === 6 && !form.isEmailVerified) {
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)